pkg/sypgp/sypgp.go

```diff
@@ -372,18 +372,37 @@ func GenKeyPair() (entity *openpgp.Entity, err error) {
 		return
 	}
 
-	fmt.Print("Generating Entity and OpenPGP Key Pair... ")
-	entity, err = openpgp.NewEntity(name, comment, email, conf)
+	pass, err := AskQuestionNoEcho("Enter encryption passphrase : ")
+	if err != nil {
+		return
+	}
+	passRetype, err := AskQuestionNoEcho("Retype your passphrase : ")
```

Comment:
Can you please look at the comment I left in the other review?

Reply:
:+1: